milestone1/src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
```

Comment:
We don't need to create a new directory, `src/App` is ok.

And also let's delete unnecessary files and logos.

![image](https://user-images.githubusercontent.com/31897434/130460817-1f614d07-be95-4174-bfd1-0846385e1bb7.png)